android/expoview/src/main/java/versioned/host/exp/exponent/modules/api/getrandomvalues/RNGetRandomValuesModule.java

```diff
@@ -0,0 +1,36 @@
+package versioned.host.exp.exponent.modules.api.getrandomvalues;
+
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+
+import android.util.Base64;
+
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.bridge.ReactContextBaseJavaModule;
+import com.facebook.react.bridge.ReactMethod;
+import com.facebook.react.bridge.Callback;
+
+public class RNGetRandomValuesModule extends ReactContextBaseJavaModule {
+
+  private final ReactApplicationContext reactContext;
+
+  public RNGetRandomValuesModule(ReactApplicationContext reactContext) {
+    super(reactContext);
+    this.reactContext = reactContext;
+  }
+
+  @Override
+  public String getName() {
+    return "RNGetRandomValues";
+  }
+
+  @ReactMethod(isBlockingSynchronousMethod = true)
+  public String getRandomBase64(int byteLength) throws NoSuchAlgorithmException {
+    byte[] data = new byte[byteLength];
+    SecureRandom random = new SecureRandom();
```

Comment:
This native code already exists in the SDK https://github.com/expo/expo/blob/master/packages/expo-random/android/src/main/java/expo/modules/random/RandomModule.java just in an async manner. Adding this duplicative code essentially just ends up being code bloat as one of the methods would be unused.

Reply:
reasoning in description of pr